### PR TITLE
feat(pci.instances): provide a refresh action

### PR DESCRIPTION
**Pull request by radireddy**
provide a refresh icon on instances datagrid which refresh the instances

MANAGER-4320

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/instances/instances.html [hunk 1]
```diff
@@ -221,6 +222,13 @@
                     data-translate="pci_projects_project_instances_add_label"
                 ></span>
             </a>
+            <button
+                class="ml-2 float-right oui-button oui-button_secondary oui-button_icon-only oui-button_small-width"
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/instances/instances.html]: You can add `type="button"`.

**Comment by antleblanc** [comment]
@radireddy There is a typo in your commit message and also the DCO check fails.

I suggest the following message commit:

```txt
feat(pci.instances): provide a refresh action
```

Could you please have a look?

Thanks!

File: packages/manager/modules/pci/src/projects/project/instances/instances.routing.js [hunk 2]
```diff
@@ -2,23 +2,39 @@ export default /* @ngInject */ ($stateProvider) => {
   $stateProvider.state('pci.projects.project.instances', {
     url: '/instances?help',
     component: 'pciProjectsProjectInstances',
-    redirectTo: (transition) =>
-      transition
+    redirectTo: (transition) => {
+      return transition
```
Comment (JeremyDec) [inline packages/manager/modules/pci/src/projects/project/instances/instances.routing.js]: Why replacing the direct return shortcut ? (Explicit use of `return` ?)

File: packages/manager/modules/pci/src/projects/project/instances/instances.html [hunk 3]
```diff
@@ -221,6 +222,14 @@
                     data-translate="pci_projects_project_instances_add_label"
                 ></span>
             </a>
+            <button
+                type="button"
+                class="ml-2 float-right oui-button oui-button_secondary oui-button_icon-only oui-button_small-width"
+                aria-label="{{:: 'pci_projects_project_instances_refresh' | translate }}"
+                data-ng-click="$ctrl.refreshInstances()"
+            >
+                <span class="fa fa-refresh" aria-hidden="true"></span>
```
Comment (marie-j) [inline packages/manager/modules/pci/src/projects/project/instances/instances.html]: Isn't there a `oui-icon` for this ?

Comment (radireddy) [reply packages/manager/modules/pci/src/projects/project/instances/instances.html]: there is one, but the icon appears very very mall. Its the extra thin icon

**Pull request merged**